SwiftWisdom/Core/DirectoryManager/DirectoryManager.swift

```diff
@@ -11,7 +11,10 @@ import Foundation
 public final class DirectoryManager {
     
     enum Error : ErrorType {
-        case UnableToCreatePath(url: NSURL?) // TODO: When upgrading to 2.3 Swift, this type changed from NSURL -> NSURL?. Review to see if this is an appropriate change.
+        // TODO: When upgrading to 2.3 Swift, this type changed from NSURL -> NSURL?. 
```

Comment:
In the end it didn't end up getting shipped - so we have a bit more breathing room now, but not enough to look into separate error states and to create the appropriate test case scenarios for them - so either someone picks up on this thread or we can ship with the TODO. Maybe the wording should be refined?